Xamarin.Forms.Platform.Android/Resources/drawable/MaterialProgressBar.xml

```diff
@@ -0,0 +1,51 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+From original drawable https://android.googlesource.com/platform/frameworks/base/+/master/core/res/res/drawable/progress_horizontal.xml 
+with removed rounded corners.
```

Comment:
Maybe use the AppCompat - if possible - and use this style: https://github.com/aosp-mirror/platform_frameworks_base/blob/master/core/res/res/drawable/progress_horizontal_material.xml